_opcodes/op192-bgee.html

```diff
@@ -0,0 +1,14 @@
+---
+n: 192
+name: "Spell Effect: Find Familiar"
+param1: "Unused"
+param2: "Unused"
+bg1: 0
+bg2: 0
+bgee: 1
+iwd1: 0
+iwd2: 0
+pst: 0
+---
+Applies <a href="#op195">opcode #195</a>, <a href="#op196">opcode #196</a>, and <a href="#op268">opcode #268</a> to the summoned familiar.<br/>
+Familiar to be determined by <a href="../files/2da/2da_bgee/familiar.htm">FAMILIAR.2DA</a> and caster's alignment.
```

Comment:
This is the same in bg2, so just rather merge and add a note that in EEs, the mapping has been externalized to a table.

Reply:
Done.

Reply:
You still left the file in.